src/net/sourceforge/kolmafia/persistence/EquipmentDatabase.java

```diff
@@ -771,38 +771,23 @@ private static int derivePulverization(final int id) {
 
     int pulver = PULVERIZE_BITS | ELEM_TWINKLY;
     Modifiers mods = ModifierDatabase.getItemModifiers(id);
-    if (mods == null) { // Apparently no enchantments at all, which would imply that this
-      // item pulverizes to useless powder.  However, there are many items
-      // with enchantments that don't correspond to a KoLmafia modifier
-      // (the "They do nothing!" enchantment of beer goggles, for example),
-      // so this can't safely be assumed, so for now all truly unenchanted
-      // items will have to be explicitly listed in pulverize.txt.
-      pulver |= EquipmentDatabase.ELEM_TWINKLY;
-    } else {
-      for (var implication : IMPLICATIONS.entrySet()) {
-        if (mods.getDouble(implication.getKey()) > 0.0f) {
-          pulver |= implication.getValue();
-        }
-      }
-    }
 
-    int power = EquipmentDatabase.getPower(id);
-    if (power <= 0) {
-      // power is unknown, derive from requirement (which isn't always accurate)
-      pulver |= YIELD_UNCERTAIN;
-      String req = EquipmentDatabase.statRequirements.get(id);
+    // Null mods might suggest no enchantments at all, which would imply
+    // that this pulverizes to uselgiess powder.  However, there are many items
```

Comment:
```suggestion
    // that this pulverizes to useless powder.  However, there are many items
```
how did that happen?

Reply:
Since becoming a parent of two I am becoming my mother and constantly typoing basic words